docs/dvc.md

```diff
@@ -0,0 +1,62 @@
+# Data Version Control
+
+Machine Learning projects require both code and data to build a model. Both the code and the data needs to be version controlled to ensure that results are reproducible at any point in time. However, data presents two version control challenges:
+
+1. Large datasets - often the training dataset is too large to fit in a Git repository.
+2. Slow pipeline stages - Training a model on a large dataset can take a significant amount of time. It's painful to be forced to retrain a model if the training data, code, and outputs are up-to-date already.
+
+Data Version Control ([DVC](https://dvc.org/)) is a Python package that solves the above ML-specific project challenges and is installed as a dependency within whisk projects. It's a lightweight, flexible tool that aligns closely with git commands.
+
+## Enabling DVC
+
+Type [`whisk dvc setup`](cli_reference.html#whisk-dvc-setup) to enable DVC within your project. After setup completes, you need a place to store your data. whisk includes a set of commands to automate the setup of an S3 [DVC remote](https://dvc.org/doc/command-reference/remote). To create a remote:
+
+```
+whisk s3 create
+whisk dvc remote-add
+```
+
+This creates an Amazon S3 bucket named `whisk-<project_name>` and creates a DVC remote that uses the S3 bucket.
+
+## Using DVC
+
+The DVC website has [excellent documentation](https://dvc.org/doc) on using DVC. Please refer to their website.
```

Comment:
Once I try my first DVC project from scratch, I might add a couple "first steps" here on how to use DVC to help folks get their bearings. Im worried that someone might not know where to start... since `whisk dvc` commands do some things, but not others.

For example, maybe we can add something simple such as adding a new file.

Reply:
Sounds good.  Also, remember DVC isn't a requirement and isn't enabled by default. There is no requirement to use it.

If we write out some first steps, need to on top of DVC 1.0 release changes and the date: https://dvc.org/blog/dvc-3-years-and-1-0-release. I wasn't sure it's worth the effort writing out instructions prior to 1.0.

Reply:
Ah yeah that makes sense to wait until 1.0!